_posts/2021-11-01-demo-day-21-og_core.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Demo Day: The OG-Core Platform'
+author: "Jason DeBacker"
+description: The OG-Core Platform
+categories:
+- demo-days
+- python
+- macroeconomics
+- overlapping-generations
+layout: post
+toc: true
+---
+
+{% include youtube.html content='https://youtu.be/y1vSgmEtgxs' %}
+---
+
+The [OG-Core](https://pslmodels.github.io/OG-Core/content/intro/intro.html) model is a general equilibrium, overlapping generations (OG) model suitable for evaluating fiscal policy.
```

Comment:
@jdebacker. I would change this URL to just `https://pslmodels.github.io/OG-Core/`.